apps/web/src/server/applications/case/documentation/applications-documentation.guard.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Redirect to relevant reps section if folderName is relevant-representations
+ *
+ *  @type {import('express').RequestHandler<{folderName: string, caseId: string}>}
+ */
+export const assertFolderIsNotReps = (request, response, next) => {
+	if (request.params.folderName === 'relevant-representations') {
+		return response.redirect(
+			`/applications-service/case/${request.params.caseId}/relevant-representations`
+		);
+	}
+
+	next();
+};
```

Comment:
not related to the task. Just some code from a RelevantReps ticket that I moved to a more appropriate place (@DavidJWhiteDev )